### PR TITLE
react: adding missing definition for "children" in props

### PR DESCRIPTION
**Pull request by Kerryliu**
Fixes https://github.com/microsoft/griffel/issues/214

### CODE REVIEW

**Comment by Kerryliu** [comment]
@microsoft-github-policy-service agree [company="{Microsoft}"]

**Comment by Kerryliu** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: change/@griffel-react-eb078e75-469e-4056-b2e9-7293b007f909.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Adding children to prop type definitions",
```
Comment (ling1726) [inline change/@griffel-react-eb078e75-469e-4056-b2e9-7293b007f909.json:3]:
```suggestion
  "comment": "fix: Adding children to prop type definitions for React 18",
```

**Review by ling1726 — APPROVED** [review]

**Comment by layershifter** [comment]
> Related issue: #214

@Kerryliu FYI: you may want to use `Fixes #214` in such cases, https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue 😉

**Pull request merged**